earthquakes.py

```diff
@@ -27,32 +28,56 @@ def get_data():
     # To understand the structure of this text, you may want to save it
     # to a file and open it in VS Code or a browser.
     # See the README file for more information.
-    ...
+    
+    with open("earthquakes.json", "w") as target:
+        target.write(response.text)
+    
+    with open("earthquakes.json", "r") as json_file:
+        my_data_as_string = json_file.read()
+    
+    mydata = json.loads(my_data_as_string)
 
     # We need to interpret the text to get values that we can work with.
     # What format is the text in? How can we load the values?
-    return ...
+    
+    return mydata
 
 def count_earthquakes(data):
     """Get the total number of earthquakes in the response."""
-    return ...
-
+    quake_count = data["metadata"]["count"]
+    
+    return quake_count
 
 def get_magnitude(earthquake):
     """Retrive the magnitude of an earthquake item."""
-    return ...
-
+    magnitude = earthquake["properties"]["mag"]
+    
+    return magnitude
 
 def get_location(earthquake):
     """Retrieve the latitude and longitude of an earthquake item."""
     # There are three coordinates, but we don't care about the third (altitude)
-    return ...
-
+    x, y, z = earthquake["geometry"]["coordinates"]
+    
+    return f"coordinates: ({x}, {y})"
 
 def get_maximum(data):
     """Get the magnitude and location of the strongest earthquake in the data."""
-    ...
-
+    
+    featured_quakes = data["features"]
+    
+    strongest_quake = featured_quakes[0]
+    max_magnitude = get_magnitude(strongest_quake)
+    max_location = get_location(strongest_quake)
+    
+    for quake in featured_quakes:
```

Comment:
Since you've already "processed" the first earthquake, you could skip it here:
```suggestion
    for quake in featured_quakes[1:]:
```